cmd/test/test_helpers.go

```diff
@@ -44,18 +44,15 @@ func createConfig(t *testing.T) func() {
 		initTestConfig(t)
 	}
 	return func() {
-		err := os.Remove(configPath)
+		viper.SetDefault("crashreporting", false)
+		viper.SetDefault("baseurl", "holepunch.io")
+		viper.SetDefault("sshendpoint", "")
```

Comment:
this does need a default doesn't it?

Reply:
These tests werent testing tunnel yet so there wasnt a need to